res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -188,6 +188,13 @@ var PADcolors = {
     "purple": 127
 };
 
+// Two modes for the library controls
+const libraryModes = {
+    focus: 0,
+    classic: 1,
+};
+const LibraryMode = libraryModes[engine.getSetting("libraryMode")] || libraryModes.focus;
+
```

Comment:
you can remove this part entirely if you do the below instead

Reply:
What's the benefit of querying the settings each time compared to this const?

Reply:
simplicity. As long as its not done in a tight loop, querying the setting should not be expensive. Doing all of this mapping is unnecessarily weird IMO. Additionally, global `const`s in mappings are kind off iffy (can suddenly break other things due to the way JS modules work, so thats why its always been encouraged to only add properties to the controller object instead of the global one).

Reply:
Hmokay.
tight loop = called at high frequency?

So `NumarkMixtrack3.libraryMode = engine.getSetting()..` would be okay?
And settings is not an object of NumarkMixtrack3 here, right?